packages/client/src/utils/service-worker/service-worker.ts

```diff
@@ -0,0 +1,96 @@
+import { cleanupOutdatedCaches, precacheAndRoute } from 'workbox-precaching';
+
+export {};
```

Comment:
Для чего нужен пустой экспорт?

Reply:
В начале когда писал воркер не было импортов и ловил ошибку `event the error TS1208: 'service-worker.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module`, закрыл ее пустым экспортом. Сейчас он не нужен, убрал.